tests/__init__.py

```diff
@@ -72,34 +72,30 @@ def libevdev_event_nodes(self, fd_event_nodes):
 
         return devices
 
-    def catch_evdev_events(self, device, evdev_devices, callback, wait=1):
-        evs = []
+    @pytest.fixture()
+    def event_data(self, libevdev_event_nodes):
+        received = EventData()
         def collect_events(stop):  # noqa: 306
-            nonlocal evs
+            nonlocal received
             while not stop.is_set():
-                for evdev_device in evdev_devices:
-                    evs += list(evdev_device.events())
+                for evdev_device in libevdev_event_nodes:
+                    for e in evdev_device.events():
+                        if e.matches(libevdev.EV_REL.REL_X):
+                            received.x += e.value
+                        elif e.matches(libevdev.EV_REL.REL_Y):
+                            received.y += e.value
+                sleep(0.001)
```

Comment:
I pushed this `sleep` call on master already

Reply:
Okay.